docs/requirements.txt

```diff
@@ -1 +1,2 @@
 breathe
+# git+git://github.com/michaeljones/breathe@v4.3.1#egg=breathe
```

Comment:
Yes. Over winter break I plan to get the templates working and wanted to leave that there in case I forgot how to do it or if somebody else wanted to give it a go. But it can definitely be deleted, I doubt I'll forget...

Testing seemed to indicate that it was reinstalled every time the docs were built so it should probably just be removed forever